### PR TITLE
GFP analysis fix

### PR DESCRIPTION
**Pull request by ElArkk**
## PR Description

I re-ran the GFP analysis using the correct mutants. Also created a dedicated conda env to be used for the notebook.

### General
1. [x] I have made the PR off a new branch from my fork 
   (`<your_username>`:`<feature-branch_name>`), *not* 
   `<your_username>`:`master`.
2. [ ] I have added my changes to the `CHANGELOG.md` file at the top.
3. [ ] I have made any necessary changes to the documentation in the `README`.

### Code checks
1. [ ] If there are new features implemented, add suitable tests
   in the `tests` directory.
2. [x] If any new dependencies are introduced through the new features,
   add the packages, pinned to a version, to `environment.yml`.
3. [ ] Run `make test` in a console in the top level directory
   to make sure all the tests pass.
4. [ ] Run `make format` in a console in the top level directory
   to make the code comply with the formatting standards.

### CODE REVIEW

File: binder/gfp_env.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - jupyter=1.0.0
+  - jupyterlab=2.2.0
+  - matplotlib=3.1.0
+  - numpy=1.17.5
+  - optuna=1.1.0
+  - pandas=1.0.5
+  - scikit-learn=0.22.1
+  - seaborn=0.10.1
+  - pip
+
```
Comment (ericmjl) [inline binder/gfp_env.yml:19]:
```suggestion
```

Comment (ericmjl) [reply binder/gfp_env.yml:19]: To clarify, the line break isn't needed, that's all :smile:.

**Comment by ericmjl** [comment]
DAT BUTTON HAS BEEN HIT!

**Comment by ElArkk** [comment]
Ah, I was still in the process of getting the notebook to work within binder's memory constraints. Also the readme has not yet been adapted. Going to open another PR for that then :)

**Comment by ericmjl** [comment]
Whoops! My bad, next time I’ll just let you hit haha.

**Pull request merged**